3-reveal/jobs/materialized-views/MDA/MDA-Lite/reports.sql

```diff
@@ -1,6 +1,7 @@
 SELECT refresh_mat_view('kenya_mda_lite_structures', FALSE);
 SELECT refresh_mat_view('kenya_mda_lite_operational_areas', TRUE);
+SELECT refresh_mat_view('mda_lite_plans', TRUE);
+SELECT refresh_mat_view('kenya_mda_lite_jurisdictions', TRUE);
 SELECT refresh_mat_view('kenya_mda_lite_wards', TRUE);
 SELECT refresh_mat_view('supervisor_drug_distribution', TRUE);
-SELECT refresh_mat_view('mda_lite_plans', TRUE);
 SELECT refresh_mat_view('ward_drug_distribution', TRUE);
```

Comment:
Of these, which ones are exposed to Superset slices?

Reply:
Only the `kenya_mda_lite_structures` (the first one) is not exposed. All the others have slices.